src/modules/Project/components/AddProjectDialog.tsx

```diff
@@ -113,9 +118,13 @@ export function AddProjectDialog({
           <Button onClick={() => handleClose()}>Cancel</Button>
           <Button
             onClick={() => {
+              trigger();
               if (formState.isValid) {
                 addProjectForm.current!.submit();
                 additionalAction();
+              } else {
+                const values = getValues();
+                createProject(values);
```

Comment:
Do I understand correctly, that you create a project only when the form is invalid?

Reply:
Lol :D tnx for pay my attention. I have to change it.